de.metas.contracts/src/main/sql/postgresql/system/50-de.metas.contracts/5488820_sys_gh3718_AddExtensonTypeList.sql

```diff
@@ -0,0 +1,144 @@
+;
+
+-- 2018-03-19T17:27:44.481
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Reference (AD_Client_ID,AD_Org_ID,AD_Reference_ID,Created,CreatedBy,EntityType,IsActive,IsOrderByValue,Name,Updated,UpdatedBy,ValidationType) VALUES (0,0,540843,TO_TIMESTAMP('2018-03-19 17:27:44','YYYY-MM-DD HH24:MI:SS'),100,'de.metas.contracts','Y','N','ExtensionType',TO_TIMESTAMP('2018-03-19 17:27:44','YYYY-MM-DD HH24:MI:SS'),100,'L')
+;
+
+-- 2018-03-19T17:27:44.493
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Reference_Trl (AD_Language,AD_Reference_ID, Description,Help,Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy) SELECT l.AD_Language,t.AD_Reference_ID, t.Description,t.Help,t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy FROM AD_Language l, AD_Reference t WHERE l.IsActive='Y' AND l.IsSystemLanguage='Y' AND l.IsBaseLanguage='N' AND t.AD_Reference_ID=540843 AND NOT EXISTS (SELECT 1 FROM AD_Reference_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Reference_ID=t.AD_Reference_ID)
+;
+
+-- 2018-03-19T17:36:43.631
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Ref_List (AD_Client_ID,AD_Org_ID,AD_Ref_List_ID,AD_Reference_ID,Created,CreatedBy,EntityType,IsActive,Name,Updated,UpdatedBy,Value,ValueName) VALUES (0,0,541602,540843,TO_TIMESTAMP('2018-03-19 17:36:43','YYYY-MM-DD HH24:MI:SS'),100,'de.metas.contracts','Y','Extend contract for all periods',TO_TIMESTAMP('2018-03-19 17:36:43','YYYY-MM-DD HH24:MI:SS'),100,'EA','ExtendAll')
+;
+
+-- 2018-03-19T17:36:43.637
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Ref_List_Trl (AD_Language,AD_Ref_List_ID, Description,Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy) SELECT l.AD_Language,t.AD_Ref_List_ID, t.Description,t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy FROM AD_Language l, AD_Ref_List t WHERE l.IsActive='Y' AND l.IsSystemLanguage='Y' AND l.IsBaseLanguage='N' AND t.AD_Ref_List_ID=541602 AND NOT EXISTS (SELECT 1 FROM AD_Ref_List_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Ref_List_ID=t.AD_Ref_List_ID)
+;
+
+-- 2018-03-19T17:37:27.314
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Ref_List (AD_Client_ID,AD_Org_ID,AD_Ref_List_ID,AD_Reference_ID,Created,CreatedBy,EntityType,IsActive,Name,Updated,UpdatedBy,Value,ValueName) VALUES (0,0,541603,540843,TO_TIMESTAMP('2018-03-19 17:37:27','YYYY-MM-DD HH24:MI:SS'),100,'de.metas.contracts','Y','Extend contract for first period',TO_TIMESTAMP('2018-03-19 17:37:27','YYYY-MM-DD HH24:MI:SS'),100,'EO','ExtendOne')
```

Comment:
pls change the description to something like 'Extend the contract when the notice date is reached'
..because afaiu that's what it does, right?